docs/3.0.0/docs/canton/usermanual/pruning.rst

```diff
@@ -43,6 +43,59 @@ For the maximum duration to specify a reliable pruning window "end time", the le
 should should not be wildcards (`*`) as illustrated in the examples above. If the hour field is fixed, so should the
 fields for minute and second.
 
+Schedule Format
+---------------
+
+The format used to describe cron expression consists of 7 whitespace-separated fields:
+
+1. seconds
+2. minutes
+3. hours
+4. day-of-month
+5. month
+6. day-of-week
+7. year (optional)
+
+Each field has a set of valid values:
+* seconds and minutes must be a number between ``0`` and ``59``
+* hours must be a number between ``0`` and ``23``
+* day of the month must be a number between ``1`` and ``31``
+* month must be either
+  * a number between ``0`` and ``11`` (with ``0`` being January), or
+  * the first three letters of the month name (``JAN``, ``FEB``, ``MAR``, ...)
+* day of the week must be either
+  * a number between ``1`` and ``7`` (with ``11`` being Sunday), or
```

Comment:
I think there's an extra `1` here (probably habituated from all the double-backtics 😄 )

```suggestion
  * a number between ``1`` and ``7`` (with ``1`` being Sunday), or
```